src/s2e/Plugins/OSMonitors/Support/ModuleMap.h

```diff
@@ -19,10 +20,37 @@
 namespace s2e {
 namespace plugins {
 
+typedef enum S2E_MODULE_MAP_COMMANDS { GET_MODULE_INFO } S2E_MODULE_MAP_COMMANDS;
```

Comment:
See my comment in https://github.com/S2E/guest-tools/pull/17 asking if we can remove this code duplication by reusing the header file.

Reply:
Can you at least add a comment here pointing out the duplication with guest-tools and if one is changed that other must be changed as well.

Reply:
Yes. I also created an issue to refactor that.
https://github.com/S2E/s2e-env/issues/96